packages/angular/src/sdk.ts

```diff
@@ -33,6 +33,7 @@ export function getDefaultIntegrations(_options: BrowserOptions = {}): Integrati
   //  - https://github.com/getsentry/sentry-javascript/issues/5417#issuecomment-1453407097
   //  - https://github.com/getsentry/sentry-javascript/issues/2744
   return [
+    // TODO(v10): Replace with `eventFiltersIntegration` once we remove the deprecated `inboundFiltersIntegration`
     inboundFiltersIntegration(),
```

Comment:
Wherever we currently add `inboundFiltersIntegration` as a default integration, we can't simply swap it out. This is because it would break people filtering out the integration by its name. IMHO it's perfectly fine to keep this until the next major (or whenever we decide to remove the old name export) and then make the swap.